airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/manifest.yaml

```diff
@@ -38,9 +33,15 @@ definitions:
         - type: DefaultErrorHandler
           backoff_strategies:
             - type: ConstantBackoffStrategy
-              backoff_time_in_seconds: 300
+              backoff_time_in_seconds: 100
```

Comment:
was 300 just excessive?

Reply:
Yes. I did several tests and 100 is more than enough.